docs/decex/fraud_proofs.md

```diff
@@ -8,21 +8,32 @@ keywords:
     - fraud proof
     - challenge period
 last_update:
```

Comment:
maybe we should have a changelog that can retain what has been updated instead. Not a blocker just a thought

Reply:
I don't think markdown supports that. There's always git blame though